modules/crypt.cpp

```diff
@@ -45,12 +45,24 @@
 #include <znc/IRCNetwork.h>
 
 #define REQUIRESSL 1
-#define NICK_PREFIX_KEY "[nick-prefix]"
+#define NICK_PREFIX_KEY "@nick-prefix@"
```

Comment:
Could add code to the NickPrefix() function that first checks NICK_PREFIX_KEY (with @), if it doesn't find that to check NICK_PREFIX_KEY_OLD (with []) and to set that value under NICK_PREFIX_KEY then?